cmd/fyne_settings/settings/appearance.go

```diff
@@ -11,14 +11,18 @@ import (
 	"fyne.io/fyne/v2/app"
 	"fyne.io/fyne/v2/canvas"
 	"fyne.io/fyne/v2/container"
+	internalapp "fyne.io/fyne/v2/internal/app"
 	intWidget "fyne.io/fyne/v2/internal/widget"
 	"fyne.io/fyne/v2/layout"
 	"fyne.io/fyne/v2/theme"
 	"fyne.io/fyne/v2/widget"
 )
 
 const (
-	systemThemeName = "system default"
+	themeNameDark           = "dark"
+	themeNameLight          = "light"
+	themeNameSystemLabel    = "system default"
+	themeNameSystemSettings = ""
```

Comment:
I was a little confused by the naming - would `themeNameSystem` be OK? It feels like that matches the other keys, leaving the `Label` variant just used for label purposes?